package.json

```diff
@@ -48,7 +48,7 @@
     "react-ga": "^2.5.5",
     "react-helmet": "^5.2.0",
     "react-markdown": "^4.0.3",
-    "react-redux": "^5.1.1",
+    "react-redux": "^7.1.1",
```

Comment:
> refactored navbar styling and logo spacings 

I think we should do such upgrades via the separate PRs due to high chance of breaking something (2 major versions is not a joke)

Reply:
I looked over the changelogs, doesn't seem to be anything breaking that we use: https://github.com/reduxjs/react-redux/releases